### ISSUE TITLE
Suggestions List - Open to hear opinions from others!

### ISSUE THREAD

**Issue by BulletPR00FXX** [xJon/Hexxit-II#114]
Hey Jon! Would firstly like to thank you for your hard work on this great pack!  But i have a few suggestions for you to review and possibly roll out in future updates!  These suggestions are all personal opinion, but I believe they would be great additions to the pack/theme.

1)  Stonelings should drop Hexxical essence - Stonelings (from quark mod) drop several other high value items such as gold, iron, lapis, diamonds, emeralds etc.  why not add essence and possibly even other items like fallen star pieces or waystone warp stones (we all know stonelings like shiny items like these lol)
  
2)  Better Storage Too mod - Better storage was in the original Hexxit.  Better storage is great because it has lockable chests (with enchanted locks/keys) as well as a very cool crate that can expand in size. (great for dumping items into until you have time to properly store your items into chests)

3)  Simple storage network mod - This mod is an extremely simple (and requires no power) storage mod comparable to AE2 or Refined storage.  With this mod you still have your clutter of chests, but can easily access all of their inventories at one time, making it easier to find the items your looking for.  IMO hexxit has always been full of items that cause cluttered chests, and this would be an excellent fix for it!  (just be sure that if you have better storage too as well as this mod, make sure you set the configs so that it wont link crate to the storage network that I mentioned above, as I think it causes an issue.)

4) Hexxit is the name of the modpack, yet...  Hexxical Armor sets are not the best in the pack, and this bothers me...  Either the Hexxical Sets need a buff (especially after the recent nerf/crafting recipe changes) or tinkers construct armors need to go, or both.  I'm not 100% on the best thing to do to fix this, as the world is far more dangerous than in original hexxit, and OP armor is very nice... but the end game armor is maxed out construct armory armor.  I'd rather see people running around in hexxical gear myself.

5)  Mob grinding is much different in Hexxxit II from the original.  I have made some very awesome/elaborate auto kill farms in OG hexxit, but with the new version of Reliquary, you need mob drops that are only obtainable if a player kills the mob.  I'd really like to see a mod, possibly like cyclic added that has simple blocks that can kill mobs as if killed by a player.  Cyclic is a good choice imo because it adds not only simple machines, but magical items, similar to reliquary and chocolate quest.   Cyclic would add useful things such as Block placer, Block miner, auto sheep shearing block, conveyer belts, experience pylon, spikes, automated user, and more, as well as fun items like several scepters/wands, cave finders, spawner seekers, inventory crafting table, tool, weapons, potions, enchantments, and so on.  only problem with cyclic is some items require power, so they would either need to be allowed or removed, up to you. (Or you can take the idea I used in my personal 1.16.5 pack, and add something like a solar panel hat (Solar Helmet Mod) so players have to adventure outdoors to charge their items, lol)  Another good option might be mob grinding utils, which adds redstone powered items like fans,  mob masher, conveyers, etc.  I haven't tested recently to be 100% on what will make "player kill" drops, but I think it would be nice to have a small mod to help with getting reliquary drops without having to hand kill every single mob type to get these items.

This is all that I have for now, but thank you for your time reading these!  Hope you can give them some thought, and id like to know yours as well as other fans of the modpacks opinions on these suggestions!

**Comment by BulletPR00FXX** [comment]
6)  Personally dislike the backpack mod used, and think a simpler one would be better.  Seems like most people have issues crafting them as well, due to the different tiers.  there are several other backpack mods that would be better fit, imo.

7)  I believe that the current gravestone mod could also be changed, possibly for corail tombstones?  there are others out there as well, but corail has an interesting magical aspect (items and other things) that might fit the theme decently well.  The current gravestone mod also has a lot of players confused I have noticed, as you have to crouch on the tomb to receive items.

**Comment by BulletPR00FXX** [comment]
8.  Ender Storage mod - Another QoL mod, and it fits the simple yet magical theme of hexxit. (https://www.curseforge.com/minecraft/mc-mods/ender-storage-1-8)

**Comment by BulletPR00FXX** [comment]
**UPDATE**
found an item that I was hopeful might fix issue #5, the punji stick from tinkers construct. However, it does not count as a player kill when something dyes by punji sticks. There are Far too many mobs that need killed by a player to get reliquary and other items.  It would be great to have some kind of mod added that could make mob farms a little less of a grind.

**Comment by BulletPR00FXX** [comment]
Another mod that might be good to add to fix issue 5 - Dark Utilities.  It doesnt add a lot, but would do the trick to make reliquary mob drops.
(https://www.curseforge.com/minecraft/mc-mods/dark-utilities)

**Comment by xJon** [comment]
Hey! Thank you so much for putting all the effort into making suggestions for improving the modpack :)
I'd like to address each of them:
1. This is a neat idea, and I'll take that into consideration. From what it looks, changing the Stonelings' drops is not a configurable feature that Quark offers by default in 1.12, but I'm pretty certain it'd be possible using other mods' help.
2. That's a great mod suggestion and I'm always for adding features which were in the original Hexxit. I'll make sure to test its 1.12.2 version and if it works well, look into adding it in the next modpack update!
3. This might be a recurring motif, but this mod suggestion probably falls into "not in the spirit of Hexxit/vanilla+" category. It can indeed help "fix" issues such as clattered chests, but at the same time change the feeling of the game, so for now we'll keep it out of the modpack. Thank you though also for pointing out configuring the it with Better Storage Too.
4. That's an interesting point. I'm not sure whether you can find objectively better counters to all of the passive and active abilities Hexxit Gear offers, but if you feel it should be improved, I'd love to hear in what way specifically!
5. I read everything you wrote about this issue, and from what I'm understanding the main difference is Reliquary's current requirement for player kills. I think that the easiest solution for this whilst staying in the spirit of Hexxit would be to use Reliquary's own [Pedestal block](https://github.com/P3pp3rF1y/Reliquary/wiki/Blocks#pedestal) - it can automatically kill mobs and counts as a player kill.
6. to 8. Will take all 3 of these suggestions into consideration :)

**Comment by BulletPR00FXX** [comment]
Thanks for your reply!  The original Hexxit was my all time favorite modpack, and am happy to help make Hexxit II just as awesome!  I'd also like to add that after playing for some time now, I have found that stonelings, while somewhat hard to find (my underground base became infested with them) win the game when you make a spawner.  Took me a bit to find a method, (water doesnt push them, they dont take fall damage, and if I remember correctly, they do not drown) but openblocks fans do push them, and dropping them through lava works great, as they burn to death at the bottom of my spawner.   This has given me 300k+ coal, redstone (used in tome of alkaheist) lapis, thousands of iron and gold ingots, diamonds, horse armors (to be melted into more metals) etc.  they are definitely the go-to first spawner to build imo.  Its very cool, but idk if thats something you want to keep/change or not.
And thank you for letting me know about the Pedestal Block! I did not know that it could do such a thing.  Ill try that out and if I run into issues ill return here.  Thanks again for all of your input!

**Comment by xJon** [comment]
@BulletPR00FXX That's awesome, and thank you for sharing your elaborate farming techniques! Regarding the swords in pedestals, I assume you tried mending and it does not work with them?

**Comment by BulletPR00FXX** [comment]
I had not tried mending, and it does seem to work, good thinking!  Thanks!

**Comment by BulletPR00FXX** [comment]
Well, I'm back to wishing I had a better mob grinding item.  The Reliquary Pedestal is okay...  but its very slow at killing.  It cannot keep up with the Soul Shards tier 5 spawner, not even close.  Id much rather drop mobs on spikes or something like that.  In theory, I could just add more pedestals.  while this is true, it doesnt help my next issue.  The Mending ability will not keep swords from breaking with mobs that have a high health.  The highest Damage sword I have found is the Monking Sword.  I have had one fully enchanted break extremely fast when spawning Guardians (30 hearts).

**Comment by COmptonWarlord** [comment]
/Back takes an extra long time, i know gravestones wont delete my items but i would like to get back in the action ASAP you know what im saying

**Comment by BulletPR00FXX** [comment]
> /Back takes an extra long time, i know gravestones wont delete my items but i would like to get back in the action ASAP you know what im saying

this is a plugin/server issue.  not modpack related.

**Comment by BulletPR00FXX** [comment]
>     4. That's an interesting point. I'm not sure whether you can find objectively better counters to all of the passive and active abilities Hexxit Gear offers, but if you feel it should be improved, I'd love to hear in what way specifically!

About this...   Tinkers armor can do all of the things hexxit armor sets can do, plus have extremely high resistance.  the resistance is very necessary, as there is a lot more death in hexxit 2.  Especially in the Nether.  The nether fortresses are insta death w/o tinkers armor.  Not sure on a fix, though...  Other than tweaking the hexxit armor mod itself.

**Comment by BulletPR00FXX** [comment]
> @BulletPR00FXX That's awesome, and thank you for sharing your elaborate farming techniques! Regarding the swords in pedestals, I assume you tried mending and it does not work with them?

I have decided to go with 4 tinkers unbreakable scythes.  these do are damage, and do an okay job.  i am using 4 of them in a somewhat large chamber (11x11x3) and can spawn any of up to 8 mob types in this chamber.  The pedestals swing pretty slow and do not keep up with the tier 5 soul shards very well, but it works.  on the other hand, the drops (loot and XP) are my next concern.  I have used 4 vacuum hoppers to collect the loot, and put it into drawers.  I have one drawer that hoppers into a dropper to dispose of junk into a lava source.  Mx xp collection is currently going down into 4 pedestals with heroes medalions on them.  these medalions are very slow at gathering xp (i have a few vac hoppers outputing into them).  this is causing major lag at times, if i dont collect the xp by hand.

**Comment by BulletPR00FXX** [comment]
I am now finding that my scythes are falling off pedestals.  I found one on the ground under my spawner, and upon returning it to its place, i noticed 2 others were also missing/gone forever.  Very annoying.  My guess is that one of the infernal mobs had a disarm ability.  Would be so much better if there was something else to autokill with than the annoying pedastals.

**Comment by BulletPR00FXX** [comment]
> You can try to use the Recall enchantment:
> 
> > whenever a recall enchanted item is dropped and touches the ground, it comes back into the thrower inventory after a last update.

if it will work on tinkers items, idk.  i have considered it, but its been very tedious to make this all work, now I need to remake and tool level them to add unbreaking.   very discouraging.

**Comment by BulletPR00FXX** [comment]
> You can try to use the Recall enchantment:
> 
> > whenever a recall enchanted item is dropped and touches the ground, it comes back into the thrower inventory after a last update.

UPDATE: Recall Doesnt work on tinkers tools.  Seems like only the enchanting plus table works (sometimes, but buggy) on tinkers tools, and anvil/enchanted books do not work.  only options from the enchanting plus tabe for it was soulbound and unbreaking.

**Comment by BulletPR00FXX** [comment]
And not using unbreakable tinkers tool tends to occasionally (sometimes frequently if used with stronger mobs such as guardians or stronger mobs) break anyway even when using mending sometimes.  its very annoying to try to find a way to deal with this.  also, the experience drops are slow to suck into the pedestals so its near impossible to keep your experience from a mob farm.  im currently just dumping it into lava.

**Comment by muellersebi014** [comment]
> 6\. Personally dislike the backpack mod used, and think a simpler one would be better.  Seems like most people have issues crafting them as well, due to the different tiers.  there are several other backpack mods that would be better fit, imo.

I strongly belief that this is the biggest issue with the modpack right now. So really: **Please do something about it.**

**Comment by BulletPR00FXX** [comment]
> What is the biggest issue with it, specifically? You can craft backpacks as long as the component tiers match.

component tiers are confusing to many players for one, and, they take a ton of resources (tons of sting and leather, sticks/iron sticks aren't bad) to make, making early game storage difficult.

**Comment by muellersebi014** [comment]
Summing up, the backpacks have become a nuisance to craft.

   1. The worst part is that the crafting of one backpack takes of these
   bag-crafting being split in what-feels-like-a-thousand steps:
   - "Crafting the Frames"
      - "Crafting the Top Backpack Piece"
      - "Crafting the Middle Backpack Piece"
      - "Crafting the Bottom Backpack Piece"
      - "Crafting the Backpack".

   That's 5 steps for the *small* backpacks. The big ones take an
   additional 2 Steps (crafting the tanned leather and then cooking them).
   The old small ones did it in 1 (vs 5 now) step and 3 (vs 7 now) for the
   big ones.
   Sould you make a mistake while crafting, you lose quite a lot of
   resources...

   2. They cannot be found in treasure chests (we know that's not your
   fault, but the mod's creator's).

   3. The amount of resources themselves are less of a problem to me.
   However, it feels like you get much less of them in comparison to Hexx1.
   Due to
      - leather being 4 times as expensive to make from rotten-flesh
      - the overall availability of leather resources due to Mo-Creatures
      which spawns less of leather-bringing mobs and instead spawns a
lot of big
      cats and bears.

   4. The fact that the items of the same kind don't stack (like frames and
   pieces), doesn't make it better.


Am So., 27. Feb. 2022 um 05:33 Uhr schrieb BulletPR00FXX <
***@***.***>:

**Comment by muellersebi014** [comment]
Summing up, the backpacks have become a nuisance to craft.

   1. The worst part is that the crafting of one backpack takes of these
   bag-crafting being split in what-feels-like-a-thousand steps:
   - "Crafting the Frames"
      - "Crafting the Top Backpack Piece"
      - "Crafting the Middle Backpack Piece"
      - "Crafting the Bottom Backpack Piece"
      - "Crafting the Backpack".

   That's 5 steps for the *small* backpacks. The big ones take an
   additional 2 Steps (crafting the tanned leather and then cooking them).
   The old small ones did it in 1 (vs 5 now) step and 3 (vs 7 now) for the
   big ones.
   Sould you make a mistake while crafting, you lose quite a lot of
   resources...

   2. They cannot be found in treasure chests (we know that's not your
   fault, but the mod's creator's).

   On that note: i just found out you can use

>    /give {player} backpack:backpack_piece 1 X
>    {leather_tier:"II",frame_tier:"II"}

   (X is 0,1 or 2 and represents the upper,middle,bottom part)
   to give the parts to people. Maybe you can somehow implement this into
   the lootgeneration.

   The backpacks themselves use UUIDs - therefore they will be very hard to
   be used in lootgeneration. (Good part about it: you can copy a Backpack and
   then multiple people have a shared inventory)

   3. The amount of resources themselves are less of a problem to me.
   However, it feels like you get much less of them in comparison to Hexx1.
   Due to
      - leather being 4 times as expensive to make from rotten-flesh
      - the overall availability of leather resources due to Mo-Creatures
      which spawns less of leather-bringing mobs and instead spawns a
lot of big
      cats and bears.

   4. The fact that the items of the same kind don't stack (like frames and
   pieces), doesn't make it better.


Am So., 27. Feb. 2022 um 03:31 Uhr schrieb Jon ***@***.***>:

**Comment by BulletPR00FXX** [comment]
> 

to note your note on #3, i have noticed as well that if you die, then restore your player data backup, then retrieve your lost items, of course your items are duplicated, but the items inside the duped backpacks are not, but the inventories between them are as you said, shared.  i thought this was interesting, but probably not intended.

**Comment by muellersebi014** [comment]
<details>
  <summary> Offtopic, regarding Hexxit-1 </summary>

Hexxit-1 had the same thing. Rarely you'd find Backpacks in a stack of 2. They also had a shared invenotry that worked for two players. Don't know whether it survived a server restart. 
  
</details>

**Comment by xJon** [comment]
@BulletPR00FXX I understand your frustration and I'm really sorry to hear it! I'd like to clear some things up:
You can craft a backpack with leather and 1 frame (not using multiple pieces), but they will be smaller due to the simpler crafting process.
Additionally, although you can't find full backpacks in loot chests, I made sure players will find leather, string, sticks and most importantly bound & tanned leather in loot. 
Also, in the first Hexxit, bound leather required 7 string each, a recipe which was eased in Hexxit II with 3 string each only. Cows were also made to drop more leather in Hexxit II.

To summarize, there was a lot of thought put into the balance of this game mechanic. Backpacks *are* more complicated in this version, as the highest tier of backpack requires more crafting steps and more tanned leather (14 instead of 8), but with other implemented changes, this should be pretty balanced overall. I agree that the mod's JEI integration is not ideal, but at this stage I'd rather make smaller tweaks to balance instead of replacing the mod completely. Let me know your thoughts!

**Comment by BulletPR00FXX** [comment]
> @BulletPR00FXX I understand your frustration and I'm really sorry to hear it! I'd like to clear some things up: You can craft a backpack with leather and 1 frame (not using multiple pieces), but they will be smaller due to the simpler crafting process. Additionally, although you can't find full backpacks in loot chests, I made sure players will find leather, string, sticks and most importantly bound & tanned leather in loot. Also, in the first Hexxit, bound leather required 7 string each, a recipe which was eased in Hexxit II with 3 string each only. Cows were also made to drop more leather in Hexxit II.
> 
> To summarize, there was a lot of thought put into the balance of this game mechanic. Backpacks _are_ more complicated in this version, as the highest tier of backpack requires more crafting steps and more tanned leather (14 instead of 8), but with other implemented changes, this should be pretty balanced overall. I agree that the mod's JEI integration is not ideal, but at this stage I'd rather make smaller tweaks to balance instead of replacing the mod completely. Let me know your thoughts!

My thoughts are still that the backpacks are too complicated to make, and there are better backpack mods out there.
I also still believe that the reliquary mob grinders are horrible, lol.

**Comment by KELETU66666** [comment]
Did anyone think MCM is fun? I think this mod takes up a lot of memory and is of poor quality, not as good as zulu, which is a world away from zulu...
